ci/checks/style.sh

```diff
@@ -12,7 +12,7 @@ PATH=/conda/bin:$PATH
 source activate gdf
 
 # Run flake8 and get results/return code
-FLAKE=`flake8 --exclude=cuML,ml-prims,__init__.py,versioneer.py && flake8 --config=python/.flake8.cython`
+FLAKE=`flake8 --exclude=cpp,thirdparty,__init__.py,versioneer.py && flake8 --config=python/.flake8.cython`
```

Comment:
Wasn't this already in another PR? Is it a merge issue?

Reply:
Dante's PR was failing because he needed my changes and this PR was failing because I needed his. This PR contains the commits from both Dante.